src/golang/cmd/server/server/handlers.go

```diff
@@ -15,7 +15,6 @@ func (s *AqServer) Handlers() map[string]handler.Handler {
 		routes.ConnectIntegrationRoute: &handler.ConnectIntegrationHandler{
 			Database:   s.Database,
 			JobManager: s.JobManager,
-			Vault:      s.Vault,
```

Comment:
Can we remove the vault object from the server, that way we prevent any misuse in the future.

Reply:
Seems that we have a lot of other handlers currently making use of this `s.Vault`. Are these all bugs?